app/controllers/application_controller.rb

```diff
@@ -1872,6 +1872,10 @@ def remember_tab
 
     url = URI.parse(request.url).path
 
+    if (url.include? "launch_vmware_console") || (url.include? "launch_html5_console")
```

Comment:
What do you think about?
```ruby
return if %w(launch_vmware_console launch_html5_console).any? { |i| url.include?(i) }
```
This would also solve the rubocop issues...

Reply:
Also a comment above why do we need this line would be nice.

Reply:
Thanks, I will apply your suggestions